deleteTempCache.php

```diff
@@ -0,0 +1,11 @@
+// This scipt is called by an administrator of the Formulize install within the Admin section of the site, in order to remove all files in the 'templates_c' folder.
+// This script should only be called if the user is developing new templates and needs a a quick way to delete existing cached template data.
+<?php
+    $files = glob($_SERVER["DOCUMENT_ROOT"].'/formulize/templates_c/*'); // Get all file names in directory.
```

Comment:
Rather than adding this code in the deleteTempCache.php file, this code could be moved to the formulize_xhr_responder.php file as it deals with AJAX requests, loading the mainfile, checking permissions and so on.